packages/exoflex/src/components/RadioButton.tsx

```diff
@@ -76,13 +81,22 @@ export default function RadioButton(props: RadioButtonProps) {
 
   let isChecked = contextValue === value || checked;
 
+  // NOTE: Use `button` for web as RNW doesn't support it yet
+  // https://github.com/necolas/react-native-web/blob/master/packages/react-native-web/src/modules/AccessibilityUtil/propsToAriaRole.js
+  let defaultAccessibilityRole = (IS_MOBILE ? 'radio' : 'button') as
+    | 'radio'
+    | 'button';
+
   let handlePress = () =>
     contextOnValueChange
       ? contextOnValueChange(value)
       : onPress(!isChecked, value); // NOTE: `onPress` should returns nothing, so this behaviour should be changes in v4
 
   return (
     <TouchableOpacity
+      {...otherProps}
+      accessibilityLabel={accessibilityLabel || `Radio Item: ${label}`}
```

Comment:
I'm curious if the colon (`:`) would be disruptive when the label is read or is it just not read at all?

Reply:
Oops, that's true! The VoiceOver might just read it literally as *"colon"*.

Reply:
If that's so, let's omit them.

Reply:
After I tried it on Android Emulator, the VoiceOver doesn't read the "colon" at all.

The VoiceOver said, "Radio Item: Male, Radio Button".

So the first "Radio Item: Male" is from the `accessibilityLabel` and the last "Radio Button" is coming from the `accessibilityRole` which we use `radio` for mobile.

From that discovery, I think it would be better if we only use the `label` prop for `accessibilityLabel`. But if we do that, probably it will "Male Button" on the web as we use `button` instead of `radio` for the `accessibilityRole`.

What do you think @danielsukmana?

Reply:
I think there are only two choices:
1. We create separate a11y labels for both mobile and web.
2. We won't provide default a11y labels and just let the user set it up on their own.

Reply:
For me, at this point, I think the second choice is more reasonable. We will just provide a default a11y role and state.

Reply:
"Male Button" doesn't seem to be very clear on what it is for. I agree, let's go with option 2.